### ISSUE TITLE
Batch norm layer

### ISSUE THREAD

**Issue by 1292765944** [YuwenXiong/py-R-FCN#75]
I notice that the batchnorm and scale layer all are set lr = 0.0 in [models/coco/ResNet-101/rfcn_end2end/train_agnostic_ohem.prototxt](https://github.com/Orpine/py-R-FCN/blob/master/models/coco/ResNet-101/rfcn_end2end/train_agnostic_ohem.prototxt#L33-L71). Why do not update the batch norm and scale parameters?
Best!

**Comment by liyi14** [comment]
Because it will perform better. One possible reason is that the batch_size we are using is too small to do meaningful updates.
